### ISSUE TITLE
Use the sony download mannager (SceDownload)

### ISSUE THREAD

**Issue by SilicaAndPina** [blastrock/pkgj#42]
Make use of the download manager built into the vita's firmware
the one used when you download something from PSN.

this way things can download in the background, and alot faster.

**Comment by blastrock** [comment]
It may be possible, but I can't find any documentation on this function. Do you have a link or something?

**Comment by SilicaAndPina** [comment]
i dont have any docs either, even on scedevnet theres like no info :d
guess i gotta reverse engineer it or smth

**Comment by kenshin088** [comment]
https://github.com/mmozeiko/pkgi/issues/3

It would be a two step process. Download the file. Open pkgj and let bit  decrypt. Or decrypt while downloading the temp file but that's harder to accomplish.

**Comment by blastrock** [comment]
Thanks for the link! I didn't know there was a discussion about this in pkgi.

As said there though, I don't want to sacrifice the fact that games are downloaded and decrypted in one pass.

Doesn't the official PSN shop manages to download games through the download manager and not need twice the space? I hope we'll be able to reverse engineer these APIs to implement this properly.

**Comment by SilicaAndPina** [comment]
yes, the offical PSN downloads and extracts pkg files on the fly, it would legit be perfect > - <
